src/App.scss

```diff
@@ -1,13 +1,12 @@
-
 @import './commons/assets/variables/font.scss';
 @import '././commons/assets/variables/colors.scss';
 @import '~bootstrap/scss/bootstrap';
 
 h1 {
-  color: $primary-color;
-  font-family: $lato-font;
+ color: $primary-color;
+ font-family: $lato-font;
```

Comment:
@jennizalwango kindly add `font-weight:100;` here also to have the right font.

Reply:
@jennizalwango how come you didn't respond to this?